db_auth_init.sql

```diff
@@ -0,0 +1,8 @@
+-- is calles while initalizing the database
+-- as has to be run as root user to set the permissions
+
+
+CREATE DATABASE IF NOT EXISTS openpodcast_auth;
+GRANT ALL PRIVILEGES ON openpodcast_auth.* TO 'openpodcast'@'%';
+
+-- run make init_auth_db after API is running to initialize the schema of the auth db
```

Comment:
Why can't we always create this schema? Can't it be a migration?

Reply:
nope, as the auth space in not covered by the migration and is accessed by a read only connection